engine/table/src/main/java/io/deephaven/engine/table/impl/select/python/FormulaKernelPythonSingularFunction.java

```diff
@@ -162,6 +162,6 @@ public <T> void applyFormulaChunk(
 
     @Override
     public FillContext makeFillContext(int __chunkCapacity) {
-        return FillContextPython.EMPTY;
+        return new FillContextPython();
```

Comment:
If we're going to have poisoned, unusable, empty contexts, the singleton was probably better.